### PR TITLE
Add docker-compose config

### PR DESCRIPTION
**Pull request by sergeyKhristenko**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+FROM node:18
```
Comment (dmsmaxim) [inline Dockerfile:1]: Lets use multistage with `npm prune`

[the remark this answers is not in the document]
Comment (dmsmaxim) [reply Dockerfile:1]: Maybe we can also use distroless?

[the remark this answers is not in the document]
Comment (dmsmaxim) [reply Dockerfile:1]: Or at least alpine image

[the remark this answers is not in the document]
Comment (sergeyKhristenko) [reply Dockerfile]: I guess node-slim should be enough for now.

**Pull request merged**